SoftwareGuide/Latex/DevelopmentGuidelines/WriteAFilter.tex

```diff
@@ -478,6 +478,13 @@ \section{Threaded Filter Execution}
 If more than one thread tries to write to the same place at the same time,
 the program can behave badly, and possibly even deadlock or crash.
 
+In ITK version 5.0, `DynamicThreadedGenerateData` signature was introduced.
```

Comment:
We should recommend using `DynamicThreadedGenerateData` by default and not discuss `ThreadedGenerateData` to keep things simple.